### ISSUE TITLE
http.IncomingMessage is undefined

### ISSUE THREAD

**Issue by De-Lac** [jhiesey/stream-http#81]
Even using stream-http with browserify, I still got undefined error on http.IncoingMessage.

This is my code:

```
var http = require('stream-http');
var req = Object.create(http.IncomingMessage.prototype)
```

`TypeError: undefined is not an object (evaluating 'http.IncomingMessage.prototype')`

**Comment by jhiesey** [comment]
Thanks for reporting this. Fixed version coming in version 2.8.0

**Comment by jhiesey** [comment]
Fixed in https://github.com/jhiesey/stream-http/commit/bf580d32ddf792c568b2b80daa3080f37bfa9ada

**Comment by EricRabil** [comment]
This does not appear to be resolved. `http.IncomingMessage.prototype` is still undefined in v2.8.0

**Comment by jhiesey** [comment]
Whoops, sorry about that. I'll take a look tomorrow.

**Comment by jhiesey** [comment]
This time I added some tests so hopefully it's right! Will publish as soon as Travis tests run.

**Comment by jhiesey** [comment]
Published as v2.8.1
